src/docs/Contributing.stories.mdx

```diff
@@ -0,0 +1,35 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Contributing"/>
+
+# Contribution Guidelines
```

Comment:
Hey @abhilipsasahoo03, this can be proven misleading as we are only mentioning 2 projects here. Can we please change this to something better?

Reply:
Hi @Janvi-Thakkar can you please expand on which part you're referring to in the `Contributing.stories.mdx` file?